apisix/plugins/jwt-auth.lua

```diff
@@ -104,7 +105,7 @@ do
         core.table.clear(consumer_names)
 
         for _, consumer in ipairs(consumers.nodes) do
-            core.log.info("consumer node: ", core.json.delay_encode(consumer))
+            log.info("consumer node: ", core.json.delay_encode(consumer))
```

Comment:
The original `core.log` works well. Please don't touch unrelative code.

Reply:
OK.